### DISCUSSION TITLE
Temp sensor without i2c

### DISCUSSION THREAD

**Discussion by louichb22** [kizniche/Mycodo#1355] [Q&A]
I'm wondering if it's possible to connect a non-i2c temperature probe to Mycodo with a rpi4. I have a standard 5v temp probe laying around and can't seem to find a way to use it as an input in Mycodo..? If it's not possible I'll just order an i2c temp probe.. Thanks!

**Comment by mickeprag** [comment]
Late to the party. But unfortunately the rpi does not have any analog inputs so you need a ADC chip as well for your temp probe.
